src/main/resources/application.yaml

```diff
@@ -18,7 +18,7 @@ spring:
     name: Send Letter Provider
   datasource:
       url: jdbc:postgresql://${LETTER_TRACKING_DB_HOST:send-letter-producer-database}:${LETTER_TRACKING_DB_PORT:5432}/${LETTER_TRACKING_DB_NAME:letter_tracking}
-      username: letterproducer
+      username: ${LETTER_TRACKING_DB_USER_NAME:letterproducer}
```

Comment:
isn't it usually the `USERNAME`? picky question

Reply:
The postgres module gives us this value as `user_name` - I followed that